html/dom/dynamic-markup-insertion/opening-the-input-stream/event-listeners.window.js

```diff
@@ -0,0 +1,42 @@
+test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe")),
+        body = frame.contentDocument.body;
+  t.add_cleanup(() => frame.remove());
+  frame.contentDocument.addEventListener("x", t.unreached_func());
+  body.addEventListener("x", t.unreached_func());
```

Comment:
Adding descriptions to unreached_funcs makes test failures much easier to diagnose.